chore: add open api spec

ismtabo opened the pull request.

ismtabo committed:
chore: add open api spec

Canario0 commented on docs/openapi.yml:
```diff
@@ -0,0 +1,156 @@
+info:
+  title: Fiti Flaguin - OpenAPI 3.0
+  description: |-
+    This API does things
```
**thought (non-blocking):** Shall we improve this descript of the api?

ismtabo replied:
We can do this when we know what we are going to do 😅

Canario0 replied:
The list of frases right ?

Canario0 commented on docs/openapi.yml:
```diff
@@ -0,0 +1,156 @@
+    This API does things
+  termsOfService: http://swagger.io/terms/
+  contact:
+    email: pablo.rb@celtiberian.es
```
**nitpick:** pls change my mail to pablo.rb@strivelabs.io. Also u are the owner of the backend. This must be set to u not to me.

ismtabo replied:
It seems that the contact accepts only a contact object with a single mail. That's why I suggest to create an organization Gmail for these kind of things.

Shall we create it? May you do it?

Canario0 replied:
Ahh I see. Pls feel free to create it. Or add your self as author since u are the owner of the back-end.

Canario0 commented on docs/openapi.yml:
```diff
@@ -0,0 +1,156 @@
+      security:
+        - bearerAuth:
+            - read:phrase
+  /phrases/{phrasesId}:
```
**issue:** I thoght we talked about doing uuids so the clients can provide the id in the put action.

ismtabo replied:
Although PUT method must be done with the identifier in as url param. Otherwise you are trying to do a nasty POST.

Canario0 replied:
yes but the name is set to order id in the get action or Im missunderstoonding something.

Canario0 replied:
Also I remember reeding number. My bad we can resolve this.

Canario0 reviewed (approved):
Just do a quick check on the get. I thing the param name should be "- name: phrasesId" and not "- name: orderId"

ismtabo committed:
chore: apply suggestions

Updated open api information:
-  contact mail in favor of company email
- api version
- get parameter name in get and delete operations
- requestBodies definitions
- methods descriptions
- request body description

ismtabo committed:
fix: mend phrases url parameters
